apps/src/templates/teacherHomepage/TeacherHomepage.jsx

```diff
@@ -29,28 +29,28 @@ const TeacherHomepage = React.createClass({
             title={i18n.teacherCommunity()}
             description={i18n.teacherCommunityDescription()}
             image="../../static/navcard-placeholder.png"
-            buttonText={i18n.connectToday()}
-            link="forum.code.org"
+            buttonText={i18n.joinCommunity()}
+            link="https://forum.code.org"
           />
           <GradientNavCard
             title={i18n.professionalLearning()}
             description={i18n.professionalLearningDescription()}
             image="../../static/navcard-placeholder.png"
-            buttonText={i18n.learnMoreCap()}
-            link="https://code.org/professional-development-workshops"
+            buttonText={i18n.learnMore()}
+            link="https://studio.code.org/my-professional-learning"
```

Comment:
These can be relative links so that you stay in your current environment (e.g. staging/test).